guides/v2.1/cloud/live/sens-data-initial.md

```diff
@@ -78,6 +78,7 @@ This procedure corresponds to step 2 in the [recommended procedure]({{ page.base
 
 The following snippet from `config.local.php` shows an example of changing the default locale to `en_GB` and changing static file optimization settings:
 
+```php
```

Comment:
The <pre> tags are the old highlighting code. It should be removed.

Reply:
Wrap the code in a php block, and place this line immediately below the block
{: .no-copy }

Reply:
**done**